zerver/openapi/zulip.yaml

```diff
@@ -4278,16 +4278,16 @@ paths:
         presence endpoint, which returns data for all active users in the
         organization, instead.
 
-        `GET {{ api_url }}/v1/users/{email}/presence`
+        `GET {{ api_url }}/v1/users/{email_or_id}/presence`
 
         See
         [Zulip's developer documentation](https://zulip.readthedocs.io/en/latest/subsystems/presence.html)
         for details on the data model for presence in Zulip.
       parameters:
-        - name: email
+        - name: email_or_id
           in: path
           description: |
-            The email address of the user whose presence you want to fetch.
+            The email address or user_id of the user whose presence you want to fetch.
```

Comment:
We need to document the changes here; search for "Changes" in this file for an example, and document in `templates/zerver/api/changelog.md` as well.

Reply:
Ok